### PR TITLE
Add beta4 spec test

### PR DESCRIPTION
**Pull request by terencechain**
# Description
Add best4 spec tests. The minimal config of churn limit has been adjusted to 32 to test scaled to max scenario 

See: https://github.com/ethereum/consensus-specs/releases/tag/v1.1.0-beta.4

### CODE REVIEW

**Comment by terencechain** [comment]
Waiting to update SHA256s

File: spectest/shared/phase0/epoch_processing/helpers.go [hunk 1]
```diff
@@ -62,8 +61,6 @@ func RunEpochOperationTest(
 		pbState, err := v1.ProtobufBeaconState(beaconState.InnerStateUnsafe())
 		require.NoError(t, err)
 		if !proto.Equal(pbState, postBeaconState) {
-			diff, _ := messagediff.PrettyDiff(beaconState.InnerStateUnsafe(), postBeaconState)
```
Comment (terencechain) [inline spectest/shared/phase0/epoch_processing/helpers.go:65]: I find this not very helpful, when it happens it usually logs out 10000 lines of proto fields and a few lines of state fields which gets lost

**Review by prestonvanloon — APPROVED** [review]

**Comment by prestonvanloon** [comment]
Deposits still failing in e2e @terencechain

**Review by prestonvanloon — APPROVED** [review]

**Comment by prestonvanloon** [comment]
This PR is causing consistent failures in long running e2e tests.

--- FAIL: TestEndToEnd_MinimalConfig/processes_deposit_validators_epoch_13 (0.01s)

**Pull request merged**